test/Hpack/ConfigSpec.hs

```diff
@@ -642,6 +645,13 @@ spec = do
         }
         )
 
+    it "allows overriding fields and takes the last value" $ do
+      withPackageConfig_ [i|
+        name: n1
+        name: n2
+        |]
+        (packageName >>> (`shouldBe` "n2"))
```

Comment:
Now that we ignore warnings for fields that start with an underscore, is this still critical?  I think it's better not to rely on that behavior.  Please remove this test if you agree.